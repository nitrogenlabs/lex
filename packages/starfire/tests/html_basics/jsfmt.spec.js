runSpec(__dirname, ["parse5"]);
