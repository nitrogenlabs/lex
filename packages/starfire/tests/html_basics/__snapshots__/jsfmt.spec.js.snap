// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello-world.html - parse5-verify: hello-world.html 1`] = `
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0 width=device-width, initial-scale=1.0 width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Document</title>
</head>
<body>
    <!-- A comment -->
    <h1>Hello World</h1>
</body>
</html>

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">

  <meta name="viewport"content="width=device-width, initial-scale=1.0 width=device-width, initial-scale=1.0 width=device-width, initial-scale=1.0">

  <meta http-equiv="X-UA-Compatible"content="ie=edge">

  <title>Document</title>
</head>

<body><!-- A comment -->
  <h1>Hello World</h1>
</body>

</html>

`;

exports[`html-comments.html - parse5-verify: html-comments.html 1`] = `
<!-- htmlhint attr-lowercase: false -->
<html>
  <body>
    <a href="#">Anchor</a>
    <div hidden class="foo" id=bar></div>
  </body>
</html>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<!-- htmlhint attr-lowercase: false -->
<html>

<head />
<body>
  <a href="#">Anchor</a>

  <div hiddenclass="foo"id="bar" />
  </body>

</html>

`;

exports[`html-fragment.html - parse5-verify: html-fragment.html 1`] = `
<a href="#">Link</a>

<textarea>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<a href="#">Link</a>
<textarea />

`;

exports[`more-html.html - parse5-verify: more-html.html 1`] = `
<HTML>
<BODY>
    <A href="#">Anchor</A><br>
    <DIV HIDDEN CLASS="foo" IS=bar></DIV>
</BODY>
</HTMLK>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<html>

<head />
<body>
  <a href="#">Anchor</a>
<br>

  <div hiddenclass="foo"is="bar" />
  </body>

</html>

`;

exports[`void-elements.html - parse5-verify: void-elements.html 1`] = `
<html>
<head>
  <meta charset="UTF-8">
  <link rel="stylesheet" href="code-guide.css">
</head>
<body></body>
</html>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<html>

<head>
  <meta charset="UTF-8">

  <link rel="stylesheet"href="code-guide.css">
  </head>

<body></body>

</html>

`;
